src/Components/getList.js

```diff
@@ -28,8 +26,12 @@ function EmptyList() {
 }
 
 function FullList(props) {
-  const [isChecked, setCheck] = useState(false);
-
+  const [isChecked, setIsChecked] = useState(false);
+  const handleCheck = () => {
+    setIsChecked(checked => !checked);
+    // update nextPurchaseDate
+    // if checked then recalc frequency (calculateEstimate) and nextPurchaseDate (dateObj.setDate(dateObj.getDate()))
```

Comment:
This is a useful comment, but still I wonder: Is this something you still have to do? or is it explaining what line `31` does?